tasks/dependencies.rake

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+namespace :dependencies do
+  desc "Copy package.json dependencies into vendor/javascript"
+  task :vendor do
+    # Update dependencies
+    system "yarn install --frozen-lockfile"
+
```

Comment:
We are running this twice now in the test app setup which we should avoid. I think we need this in the `bin/prep-release` script instead which would be consistent with the `bin/bundle` line we run there.

```suggestion
```

Reply:
done